test/black-box/ProfileCreateCommand.test.ts

```diff
@@ -1,13 +1,10 @@
 import { rpc, api } from './lib/api';
 import { BlackBoxTestUtil } from './lib/BlackBoxTestUtil';
-import { Logger } from '../../src/core/Logger';
-import { ProfileCreateCommand } from '../../src/api/commands/profile/ProfileCreateCommand';
 
 describe('ProfileCreateCommand', () => {
 
     const testUtil = new BlackBoxTestUtil();
-    const profileService = null;
-    const method =  new ProfileCreateCommand(profileService, Logger).name;
+    const method = 'profile';
 
```

Comment:
add:
import { Commands } from '../../../src/api/commands/CommandEnumType';
and then you can:
const rootMethod = Commands.PROFILE_ROOT;